### DISCUSSION TITLE
Magix VEGAS Support and Integration

### DISCUSSION THREAD

**Discussion by blu-1337** [WyattBlue/auto-editor#595] [2. Feature Ideas]
Hello, thanks for this amazing tool. I wondered if this tool can support and integrate the video editing program Magix VEGAS.

Could this be done? 

I tried exporting as various .XML files from auto-editor and importing into Vegas already edited but to no avail.

**Comment by WyattBlue** [comment]
Excluding `clip-sequence`, there's no export for Sony Vegas/Magix Vegas. 
I am not really interested in adding this feature.

**Comment by WyattBlue** [comment]
This export is now supported. `--export resolve-fcp7`
